src/features/Home/components/ModelDetailsTable/index.tsx

```diff
@@ -0,0 +1,450 @@
+                    })
+                    .map((cell, i) => {
+                      if (i === 0) {
+                        return (
+                          <th
+                            {...cell.getCellProps()}
+                            scope="row"
+                            style={{
+                              paddingLeft: '0',
+                              borderBottom: 'auto'
+                            }}
+                            key={cell.getCellProps().key}
+                          >
+                            {cell.render('Cell') as React.ReactElement}
+                          </th>
+                        );
+                      }
+                      return (
+                        <td
+                          {...cell.getCellProps()}
+                          style={{
+                            paddingLeft: '0',
+                            borderBottom: 'auto',
+                            whiteSpace: 'normal'
+                          }}
+                          key={cell.getCellProps().key}
+                        >
+                          {cell.render('Cell') as React.ReactElement}
+                        </td>
+                      );
+                    })}
+                </tr>
+              );
+            })}
+          </tbody>
+        </UswdsTable>
+      </TopScrollContainer>
+
+      {canSearch && models.length > 10 && (
+        <TablePagination
+          gotoPage={gotoPage}
+          previousPage={previousPage}
+          nextPage={nextPage}
+          canNextPage={canNextPage}
+          pageIndex={state.pageIndex}
+          pageOptions={pageOptions}
+          canPreviousPage={canPreviousPage}
+          pageCount={pageCount}
+          pageSize={state.pageSize}
+          setPageSize={setPageSize}
+          page={[]}
+        />
+      )}
+
+      <div
+        className="usa-sr-only usa-table__announcement-region"
+        aria-live="polite"
+      >
+        {currentTableSortDescription(headerGroups[0])}
+      </div>
+    </div>
+  );
+};
+
+const RenderFilteredNameHistory = ({ names }: { names: string[] }) => {
+  const { t } = useTranslation('customHome');
+  const [isShowingAllNames, setShowAllNames] = useState(false);
+
+  const firstThreeNames = names.slice(0, 3);
+
+  return (
+    <>
+      <p className="margin-y-0 font-body-xs line-height-sans-2">
+        {t('previously')}{' '}
+        {isShowingAllNames
+          ? `${names.join(', ')}`
+          : `${firstThreeNames.join(', ')}`}
+      </p>
+      {names.length > 3 && (
+        <Button
+          unstyled
+          type="button"
+          className="margin-top-1 font-body-xs"
+          onClick={() => {
+            setShowAllNames(!isShowingAllNames);
+          }}
+        >
+          {isShowingAllNames
+            ? t('viewLess')
+            : t('viewMore', { number: `${names.length - 3}` })}
+        </Button>
+      )}
+    </>
+  );
+};
```

Comment:
This already exists in `src/features/Home/Table/index.tsx`.  We should resuse it rather than recreate